frontend/src/modules/contributor/components/edit/contributor-add.vue

```diff
@@ -0,0 +1,264 @@
+                      </div>
+                    </template>
+                  </lf-input>
+                </div>
+              </div>
+            </lf-field>
+          </article>
+        </div>
+      </lf-scroll-shadow>
+    </section>
+
+    <section class="border-t border-gray-100 py-4 px-6 gap-4 flex justify-end  z-40">
+      <lf-button type="secondary-ghost" @click="isModalOpen = false">
+        Cancel
+      </lf-button>
+      <lf-button
+        type="primary"
+        :disabled="$v.$invalid"
+        :loading="sending"
+        @click="createContributor"
+      >
+        Add person
+      </lf-button>
+    </section>
+  </lf-modal>
+</template>
+
+<script setup lang="ts">
+import LfModal from '@/ui-kit/modal/Modal.vue';
+import { computed, reactive, ref } from 'vue';
+import LfButton from '@/ui-kit/button/Button.vue';
+import LfIcon from '@/ui-kit/icon/Icon.vue';
+import AppLfSubProjectsListDropdown from '@/modules/lf/segments/components/lf-sub-projects-list-dropdown.vue';
+import LfInput from '@/ui-kit/input/Input.vue';
+import LfField from '@/ui-kit/field/Field.vue';
+import { CrowdIntegrations } from '@/integrations/integrations-config';
+import { Contributor, ContributorIdentity } from '@/modules/contributor/types/Contributor';
+import LfTooltip from '@/ui-kit/tooltip/Tooltip.vue';
+import useVuelidate from '@vuelidate/core';
+import { required } from '@vuelidate/validators';
+import LfFieldMessages from '@/ui-kit/field-messages/FieldMessages.vue';
+import LfContributorAddEmailItem from '@/modules/contributor/components/edit/add/contributor-add-email-item.vue';
+import LfScrollShadow from '@/ui-kit/scrollshadow/ScrollShadow.vue';
+import { ContributorApiService } from '@/modules/contributor/services/contributor.api.service';
+import { useRouter } from 'vue-router';
+import { storeToRefs } from 'pinia';
+import { useLfSegmentsStore } from '@/modules/lf/segments/store';
+import Message from '@/shared/message/message';
+
+const props = defineProps<{
+  modelValue: boolean,
+}>();
+
+const emit = defineEmits<{(e: 'update:modelValue', value: boolean): void }>();
+
+const isModalOpen = computed({
+  get: () => props.modelValue,
+  set: (value) => emit('update:modelValue', value),
+});
+
+const router = useRouter();
+
+const { selectedProjectGroup } = storeToRefs(useLfSegmentsStore());
+
+type ContributorAddIdentity = ContributorIdentity & {
+  image: string,
+  prefix: string,
+  placeholder: string,
+}
+
+const platformList: ContributorAddIdentity[] = Object.entries(CrowdIntegrations.memberIdentities)
+  .map(([key, config]) => ({
+    prefix: config.urlPrefix,
+    placeholder: config.placeholder,
+    image: config.image,
+    platform: key,
+    type: 'username',
+    value: '',
+    verified: true,
+    sourceId: null,
+  }));
+
+interface ContributorAddForm {
+  subproject: string;
+  name: string;
+  email: string[];
+  identities: ContributorAddIdentity[];
+}
+
+const form = reactive<ContributorAddForm>({
```

Comment:
Initially when you open the drawer you have everything disabled until the user selects the sub-project. If you select it, everything becomes enabled.
However, if you close the modal and open it after, everything is enabled but sub-project is not selected.

Perhaps when the user closes the modal, everything should be cleaned and disabled again